cinema/views.py

```diff
@@ -81,12 +109,14 @@ class MovieSessionViewSet(viewsets.ModelViewSet):
         MovieSession.objects.all()
         .select_related("movie", "cinema_hall")
         .annotate(
-            tickets_available=F("cinema_hall__rows")
-            * F("cinema_hall__seats_in_row")
-            - Count("tickets")
+            tickets_available=F("cinema_hall__rows"
+                                ) * F("cinema_hall__seats_in_row"
+                                      ) - Count("tickets")
```

Comment:
There is no need to place closing parenthesis no a new line, please return to the previous version so that there would not be a case that there is inconsistency in the presence of arguments on the same line with opening and closing parenthesis